web/src/components/Upload.vue

```diff
@@ -75,12 +75,15 @@ export default {
         progress: {},
         meta: {},
       })));
+
+      const isExcelFile = file => file.name.match(/\.xlsx?$/i);
```

Comment:
There probably isn't a real difference, but I would use `file.type` to match the mimetype instead since that's what the server is relying on.

Reply:
I added both since you cannot fully trust the mimetype in different operating systems

https://developer.mozilla.org/en-US/docs/Web/API/File/type
```Developers are advised not to rely on this property as a sole validation scheme.```